### PR TITLE
feat(form-builder): allow editing of form meta

### PR DESCRIPTION
**Pull request by StephanH90**
## Description

Allows editing of form meta (similar to the question meta).

image

### CODE REVIEW

File: packages/form-builder/addon/components/cfb-form-editor/show-advanced-btn.hbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  @onClick={{@onClick}}
+  class="uk-flex uk-flex-middle uk-margin"
+>
+  {{#if @showAdvanced}}
+    <UkIcon @icon="triangle-down" />
+  {{else}}
+    <UkIcon @icon="triangle-right" />
+  {{/if}}
+  {{t "caluma.form-builder.question.advancedSettings"}}
+</UkButton>
```
Comment (anehx) [inline packages/form-builder/addon/components/cfb-form-editor/show-advanced-btn.hbs:12]: Let's implement this as wrapper component. So the call site would be something like:

```hbs
<CfbAdvancedSettings>
  my advanced settings...
</CfbAdvancedSettings>
```

This way the component using it must not implement any toggling and toggle state.

**Review by anehx — CHANGES_REQUESTED** [review]

File: packages/form-builder/addon/components/cfb-form-editor/show-advanced-btn.hbs (hunk 1 above)
Comment (StephanH90) [reply packages/form-builder/addon/components/cfb-form-editor/show-advanced-btn.hbs]: yh i was thinking about that. I changed it.

**Comment by StephanH90** [comment]
Not sure why the pipeline fails because it works for me locally:

```bash
pnpm --filter @projectcaluma/ember-form exec ember try:one ember-lts-4.8 --skip-cleanup
1..160
# tests 160
# pass  152
# skip  8
# todo  0
# fail  0

# ok
```

**Comment by anehx** [comment]
> Not sure why the pipeline fails because it works for me locally:
> 
> ```shell
> pnpm --filter @projectcaluma/ember-form exec ember try:one ember-lts-4.8 --skip-cleanup
> 1..160
> # tests 160
> # pass  152
> # skip  8
> # todo  0
> # fail  0
> 
> # ok
> ```

Flaky test: #2514

File: packages/form-builder/addon/components/cfb-form-editor/show-advanced-btn.hbs (hunk 1 above)
Comment (anehx) [reply packages/form-builder/addon/components/cfb-form-editor/show-advanced-btn.hbs]: Nice! But let's name it something like `CfbAdvancedSettings` as it isn't just a button anymore.

**Review by anehx — APPROVED** [review]

**Comment by StephanH90** [comment]
@anehx btw I can't merge this because of the flaky test.

**Pull request merged**